docstore/drivertest/drivertest.go

```diff
@@ -1558,6 +1565,50 @@ func testUpdateQuery(t *testing.T, coll *ds.Collection) {
 	})
 }
 
+func testUpdateQueryNoRev(t *testing.T, _ Harness, coll *ds.Collection) {
+	type grade struct {
+		Name  string `docstore:"name"`
+		Grade int
+	}
+
+	ctx := context.Background()
+
+	grades := []*grade{
+		{"A", 90},
+		{"B", 95},
+		{"C", 88},
+	}
+	al := coll.Actions()
+	for _, g := range grades {
+		al.Put(g)
+	}
+	if err := al.Do(ctx); err != nil {
+		t.Fatal(err)
```

Comment:
I don't agree with you in general, but I do in this case; if you look elsewhere in this file, you'll see locally defined functions called `must`. I was prevented from writing `must` as a top-level function in this file, or I would have done so and used it here. In fact, it might be worth defining locally, since we could use it two or three times.